app/src/main/java/com/nowak/wjw/simplecompass/Event.kt

```diff
@@ -0,0 +1,29 @@
+package com.nowak.wjw.simplecompass
+
+
+/**
+ * Copied from
+ * https://medium.com/androiddevelopers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150
+ */
+class Event<out T>(private val content: T) {
+
+    var hasBeenHandled = false
+        private set // Allow external read but not write
```

Comment:
property jako private i setter wtedy będzie zbędny

Reply:
tak w ogole ta klasa dla mnie ma mały sens, funkcjonalnie ją rozumiem ale jest bez sensu napisana. JAk bym coś kopiowal to juz jakies dobre rozwiazania a nie jakis kod pisany na kolanie przez kogos.

Reply:
no ja jeszcze nie do końca potrafie odróżnić jedno od drugiego LOL